lego/api/v1.py

```diff
@@ -192,6 +194,7 @@
     r"surveys/(?P<survey_pk>\d+)/submissions", SubmissionViewSet, basename="submission"
 )
 router.register(r"tags", TagViewSet)
+router.register(r"threads", ThreadViewSet)
```

Comment:
Okay thanks for the feedback I will look into this closer, this seems like a good suggestion

Reply:
The threads only have a unique ID though which is not relative to the parent forum. Do you still think /forum/id/thread/id would make sense in this case?

Reply:
Personally yeah, I think that would make sense - or you could change to slugs so it would be something like `/forum/abahus/threads/kollektiv-paa-mollenberg`, `/forum/abahus/kollektiv-paa-mollenberg` or `/forum/abahus/threads/419`.

There are many ways to design this though - I'm just suggesting this as I think it's one of the more popular designs and thus easier to understand right away. Plus it's closer to the actual use in the frontend requiring less filtering and logic there.

Although I'd guess that the others you have requested reviews from get this, or at least my first comment on email - so if there are other thoughts on this feel free to share(:

Reply:
Okay, trying to add these but it seems we strictly use the router and not urlpatterns. Not sure if I'm allowed to change this and add some custom routes like this:
`path('<int:forum_id>/threads/', ThreadViewSet.as_view({'get': 'list', 'post': 'create'}), name='forum-threads-list'),`

Reply:
haha this is completely fine by me. I think @norbye's suggestion is good.

Reply:
I'm not sure why you need this custom routing config though

Reply:
I don't think using router.register supports custom patterns like this?